### PR TITLE
VueUI Canisters

### PR DESCRIPTION
**Pull request by MarinaGryphon**
* Makes canisters use VueUI.
* Makes labels on vui-item components optional.

### CODE REVIEW

**Comment by MarinaGryphon** [comment]
!review

File: vueui/src/components/view/machinery/atmospherics/canister.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      <vui-button icon="unlocked" :params="{toggle : 1}" :class="{selected : valveOpen}">Open</vui-button><vui-button icon="locked" :params="{toggle : 1}" :class="{selected : valveOpen}"/>
+    </vui-item>
+  </div>
+</template>
+
+<script>
+export default {
+	data() {
+		return this.$root.$data.state;
+  }
```
Comment (MarinaGryphon) [inline vueui/src/components/view/machinery/atmospherics/canister.vue:50]:
```suggestion
  },
```

**Review by Geevies — APPROVED** [review]

File: vueui/src/components/view/machinery/atmospherics/canister.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      </div>
+    </vui-item>
+
+    <vui-item label="Release Valve:">
+      <vui-button icon="unlocked" :params="{toggle : 1}" :class="{selected : valveOpen}">Open</vui-button><vui-button icon="locked" :params="{toggle : 1}" :class="{selected : valveOpen}"/>
+    </vui-item>
+  </div>
+</template>
+
+<script>
```
Comment (MarinaGryphon) [inline vueui/src/components/view/machinery/atmospherics/canister.vue:46]:
```suggestion
<script>
	import Utils from "../../../utils.js";
```

File: vueui/src/components/view/machinery/atmospherics/canister.vue [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	import Utils from "../../../utils.js";
```
Comment (MarinaGryphon) [inline vueui/src/components/view/machinery/atmospherics/canister.vue:47]:
```suggestion
import Utils from "../../../utils.js";
```

File: vueui/src/components/view/machinery/atmospherics/canister.vue [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    </vui-item>
+
+    <vui-item label="Release Valve:">
+      <vui-button icon="unlocked" :params="{toggle : 1}" :class="{selected : valveOpen}">Open</vui-button><vui-button icon="locked" :params="{toggle : 1}" :class="{selected : valveOpen}"/>
+    </vui-item>
+  </div>
+</template>
+
+<script>
+import Utils from "../../../utils.js";
```
Comment (MarinaGryphon) [inline vueui/src/components/view/machinery/atmospherics/canister.vue:47]:
```suggestion
import Utils from "../../../../utils.js";
```

**Review by skull132 — APPROVED** [review]

**Pull request merged**